charts/rating/values.yaml

```
@@ -70,6 +70,10 @@ tolerations: []
 
 affinity: {}
 
+#flag to disable readinessProbe in deployment.yaml L38
+readinessProbe:
+  enabled: false
```

```suggestion
  enabled: true
```
This should be enabled by default. Disabling probes should be a voluntary act :)